kinesis/src/main/scala/akka/stream/alpakka/kinesisfirehose/KinesisFirehoseFlowStage.scala

```diff
@@ -84,7 +84,7 @@ private[kinesisfirehose] final class KinesisFirehoseFlowStage(
           inFlight -= 1
           waitingRetries.put(retryToken, Job(attempt + 1, errors.map(_._2)))
           scheduleOnce(retryToken, backoffStrategy match {
-            case Exponential => scala.math.pow(retryBaseInMillis, attempt).toInt millis
+            case Exponential => retryBaseInMillis * scala.math.pow(2, attempt - 1).toInt millis
```

Comment:
Lets make the logic identical to the one in the `KinesisFlowStage` https://github.com/akka/alpakka/pull/989/files

Which is to remove `retryBaseInMillis` completely.